25/02/2022 05:22:00-INFO-simpleSock.py-61 l2_socket established
25/02/2022 05:22:00-WARNING-simpleSock.py-89 True
25/02/2022 05:22:27-WARNING-simpleSock.py-107 60
25/02/2022 05:22:27-WARNING-simpleSock.py-108 ['', '01', '08', '15', '22', '29']
25/02/2022 05:22:27-WARNING-simpleSock.py-109 0:00:00.049350
Traceback (most recent call last):
  File "/home/wifi/Downloads/simple/Idea-validation-ip-ccn/simpleSock.py", line 95, in <module>
    emptySocket(l2_sock)
  File "/home/wifi/Downloads/simple/Idea-validation-ip-ccn/simpleSock.py", line 21, in emptySocket
    _sck.recv(123)
OSError: [Errno 100] Network is down
